src/components/__tests__/SortPicker.ts

```diff
@@ -124,3 +124,31 @@ it('subthoughts: Desc', async () => {
 
   expect(subthoughtsOfA.map((child: HTMLElement) => child.textContent)).toMatchObject(['3', '2', '1'])
 })
+
+// it('move cursor to previous sibling', () => {
```

Comment:
We should add unit tests for each of the new sort options.

You will need to come up with a good way to mock the `updated` and `created` timestamps, since in the automated test they probably all be the same by default.